Assignment/evenodd.c

```diff
@@ -0,0 +1,23 @@
+#include<stdio.h>
+#include<conio.h>
+
+int main(){
+	int s,e,i;
+	
+	
+	printf("enter starting num : \n");
+	scanf("%d",&s);
+	printf ("enter ending num : \n");
+	scanf("%d",&e);
+	
+	for(i=s;i<=e;i++){
+		if(i%2==0){

```

Comment:
we can use i/2==0 ?

Reply:
the remainder of dividing it by 2 will be 0. Thus i%2 == 0 for any even number. You can also use if (i%2 != 0) for odd numbers.

Reply:
Thank you